features/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Used only for running tests using tox, see ../tox.ini
+FROM patroni
+
+WORKDIR $PGHOME
+USER root
+RUN apt-get update \
```

Comment:
without `gcc` and `python3-dev` I am having problems with requirements installation

```
...
Building wheel for psutil (pyproject.toml) did not run successfully.  
...
      C compiler or Python headers are not installed on this system. Try to run:                                                  
      sudo apt-get install gcc python3-dev                                                                                        
      error: command 'aarch64-linux-gnu-gcc' failed: No such file or directory 
...
```

Reply:
Adding those deps. I tested on x86. Thanks for testing on another arch!

Reply:
hmmn ok. I'm not sure how to set conditional vars based on cpu arch at this point. We could add the env var name to `passenv` so if you set it in the parent environment it gets populated. As we're calling behave via a container that would need to be passed from the calling job too. Is there a way to handle this in behave? I will have a ponder on how it might be achieved in tox.